tasks/eva_2373/extract_AF.py

```diff
@@ -0,0 +1,27 @@
+from argparse import ArgumentParser
+
+import pysam
+
+
+def process_files(vcf_files_list):
+    """Extract Basic info from all the VCF files provided to populate a tsv file"""
```

Comment:
How are you populating the tsv? with the command line command using the messages being printed?

Reply:
It was through the `print` but I've changed it to an explicit output file